howto/images/publish-instance-as-image.md

```diff
@@ -0,0 +1,114 @@
+(howto-publish-instance-as-image)=
+# Publish an instance as an image
+
+Publishing an instance as an image allows you to capture the current state of an instance, including any file system modifications, and create a reusable image from it. This is useful for creating custom base images with pre-installed software, configuration changes, or other modifications.
+
+## Publish an instance
+
+To publish an instance as an image, follow these steps:
+
+### 1. Create and prepare an instance
+
+First, create a new instance from an existing image:
+
+    amc launch --name=test0 jammy:android15:amd64
```

Comment:
Should we not specify a base image here?

```suggestion
    amc launch --name=test0
```

For the purposes of the how-to, this is probably unnecessary, right?

Reply:
Hum, yeah, true. I'm fine with either.